### PR TITLE
Standardizes using latest tag for images

### PR DESCRIPTION
**Pull request by chentex**
Depends-On: https://github.com/cloud-bulldozer/benchmark-wrapper/pull/396

Signed-off-by: Vicente Zepeda Mas <vzepedam@redhat.com>

### Description

image tags are pointing to master as default. But master tag is not being built when a new version is deployed, changing it to latest.

### CODE REVIEW

**Comment by dry923** [comment]
Updated Depends-On requirement line in description

File: roles/fs-drift/tasks/main.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@
             spec:
               containers:
                 - name: publisher-container
-                  image: quay.io/cloud-bulldozer/fs-drift:master
+                  image: quay.io/cloud-bulldozer/fs-drift:latest
```
Comment (dry923) [inline roles/fs-drift/tasks/main.yml:62]: Can we update this line to be in line w/ the others to allow you to overwrite the image. ie:
```
image: "{{ workload_args.image | quay.io/cloud-bulldozer/fs-drift:latest }}"
```

**Comment by sjug** [comment]
I think `pgbench` is setting the image via defaults?

**Comment by sjug** [comment]
Thanks, I don't think `iperf3/templates/client_store.yml.j2` is used but maybe just change that image to take the `workload_args.image` parameter as well?

Otherwise lgtm.

**Pull request merged**